src/line_launcher/mod.rs

```diff
@@ -82,6 +82,14 @@ impl LineLauncher {
             NoteOffTriggerer::new(midi_message_sender.clone(), state_mutex.clone());
         note_off_triggerer.listen();
         let mut duration_between_sixteenth_notes = DurationBetweenSixteenthNotes::new();
+        let duration_ratio = Arc::new(Mutex::new(1.0));
+        let mut midi_message_bus = Bus::new(100);
```

Comment:
I feel like I've been getting myself into trouble reaching too quickly for a "class" (ie struct + methods) abstraction for things that seem like a "thing" (specifically then running into borrowing issues) so I figured I'd leave this "inline" for the moment even though it seems like a "thing" to set up a `Bus`, add a listener to it and then tell it to start "re-broadcasting" its incoming channel messages